### ISSUE TITLE
On toots with only URLs, there is nowhere in the toot area to click to expand the toot

### ISSUE THREAD

**Issue by Cassolotl** [tootsuite/mastodon#4314]
This toot is currently in the fourth column:

![screen shot 2017-07-23 at 15 00 18](https://user-images.githubusercontent.com/6607512/28499889-be9fd322-6fb7-11e7-979f-fcde83a059a5.png)

There is nowhere to click to expand that toot and, for example, see the number of faves or boosts. The URL takes up the entire line width, clicking in the space above or below the text doesn't work, and clicking the big empty space under the avatar doesn't work.

Ideally, clicking anywhere on the toot space that isn't a link (such as under the avatar) would expand the toot.

This is a duplicate of #564, which says "the ... menu has an "expand" option which also does the job". I don't think this is enough; I would never have thought to look there, and the only reason I know that menu option is there is because I found this closed issue on the Github closed issues list. I don't understand why the issue was closed, because this is a great idea.

It is connected to #140, which (am I understanding this correctly?) made neutral areas such as the empty space underneath avatars deliberately *not* expand the toot so that people can use their keyboards for navigation. If I *have* understood this correctly I can understand this need, but I'm wondering if there's a better solution that would serve that need *and* allow people to expand toots with only URLs more easily?

* * * *

- [Yes] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [N/A] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by nightpool** [comment]
I think this is prrrrobably only a cybrespace problem because of the full
width images?

**Comment by Cassolotl** [comment]
No, I have accounts on other instances and have the same problem there.

**Comment by thiagomgd** [comment]
The flagship instance has the same problem. The clickable area exists, but is way too small

**Comment by Cassolotl** [comment]
Just wanted to pop by provide a gif of what I mean.

That this is less of an issue on glitch.social, where width of toot text is a little different, but this is how it is on mastodon.social:

![apr-27-2018 22-28-13](https://user-images.githubusercontent.com/6607512/39385970-5257e1a8-4a6a-11e8-99eb-d79731e8b94e.gif)

**Comment by deutrino** [comment]
It would be good to leave an explanation when closing issues as "not actionable."

**Comment by Cassolotl** [comment]
I agree, a comment would have been nice, so that we could have known this issue was fixed with *Adds click-able div that expands status (#10733)*!

CC @deutrino
